cardano-testnet/cardano-testnet.cabal

```diff
@@ -123,8 +124,10 @@ test-suite cardano-testnet-tests
                       , tasty-expected-failure
                       , tasty-hedgehog
                       , text
+                      , time
 
   other-modules:
+                        Spec.Cli.Alonzo.LeadershipSchedule
```

Comment:
It is possible to have a leadership schedule test that is parameterized on the era?

Reply:
Not yet.  Maybe in the future when we redesign the testnet API.

Reply:
Let's create an issue to track this. We should probably label these issues with a `cardano-tesnet` label.

Reply:
https://github.com/input-output-hk/cardano-node/issues/4133